apis/python/src/tiledbsc/tiledb_group.py

```diff
@@ -29,7 +29,14 @@ def __init__(
         """
         See the TileDBObject constructor.
         """
-        super().__init__(uri=uri, name=name, parent=parent)
+        super().__init__(
```

Comment:
This is an old bug -- this was failing to propagate non-null `soma_options` through the super-chain.